### DISCUSSION TITLE
release

### DISCUSSION THREAD

**Discussion by kianzarrin** [CitiesSkylinesMods/TMPE#1547] [General]
> @aubergine10 If this is waiting for 11.6.5.3 (as stated in sidebar) please add Do not merge label

Why? can't we just take a branch from the commit before this PR merges? Is there more stuff waiting to be merged for the 11.6.5.2 release?

**Comment by originalfoo** [comment]
Stuff for 11.6.5.2 https://github.com/CitiesSkylinesMods/TMPE/milestone/64

**Comment by krzychu124** [comment]
Argh, what a mess with releases...
I'll prepare `release/11.6.5.3` (with PR to `master` to clean up git history) with cherry-picked commits from localization PR.
@aubergine10 I need a short recap (what you think fits best) for `What's new` for the `Stable` release.
It will look weird when we release `11.6.5.2-hotfix-2` as Stable
